test/utils.sh

```diff
@@ -364,6 +364,43 @@ get_base_image() {
   echo -n "$base_image"
 }
 
+ocp_to_opm_version_mapping() {
+  # OCP version e.g. v4.12
+  OCP_VERSION=$1
+
+  if [ -z "$OCP_VERSION" ]; then
+    echo "Missing OCP_VERSION PARAMETER" >&2
+    exit 2
+  fi
+
+  case "$OCP_VERSION" in
+      # === BEGIN PINNED MAP ===
+      v0.0) echo "opm-v1.28.0" ;;
+      v4.5) echo "opm-v1.26.4" ;;
+      v4.6) echo "opm-v1.26.4" ;;
+      v4.7) echo "opm-v1.26.4" ;;
+      v4.8) echo "opm-v1.26.4" ;;
+      v4.9) echo "opm-v1.26.4" ;;
+      v4.10) echo "opm-v1.26.4" ;;
+      v4.11) echo "opm-v1.26.4" ;;
+      v4.12) echo "opm-v1.26.4" ;;
+      v4.13) echo "opm-v1.26.4" ;;
+      v4.14) echo "opm-v1.26.4" ;;
+      v4.15) echo "opm-v1.26.4" ;;
+      v4.16) echo "opm-v1.26.4" ;;
+      v4.17) echo "opm-v1.40.0" ;;
+      v4.18) echo "opm-v1.44.0" ;;
+      v4.19) echo "opm-v1.48.0" ;;
+      v4.20) echo "opm-v1.50.0" ;;
+      v4.21) echo "opm-v1.50.0" ;;
+      # === END PINNED MAP ===
+      *)
+        echo "ocp_to_opm_version_mapping: No OPM version mapping for OCP $OCP_VERSION" >&2
```

Comment:
@MichalZelenak  consider returning the latest version i.e. `opm-v1.50.0` if the ocp_version is in the format `v4.x` and the mapping isn't present. 

This would come in handy for every new OCP version we bootstrap (say v4.22). Otherwise, we would need a release and update of the konflux-test image every time a new OCP version is released.